apps/examples/stack_protection/stack_protection_main.c

```diff
@@ -26,12 +26,49 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
-#define	STACK_NORMAL_SIZE		10
+#define	STACK_NORMAL_SIZE		300
 #define	STACK_OVERFLOW_SIZE		4096
 /****************************************************************************
  * Public Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: stack_prot_memory_usage
+ ****************************************************************************/
+
+static void stack_overflow_array(void)
```

Comment:
You should cover those functions with some config. If you don't, compiler will give warnings about defined but not used.

Reply:
done